hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/CacheAwareLoadBalancer.java

```diff
@@ -160,6 +168,56 @@ private RegionInfo getRegionInfoByEncodedName(BalancerClusterState cluster, Stri
     return null;
   }
 
+  @Override
+  public void throttle(RegionPlan plan) {
+    Pair<ServerName, Float> rsRatio = this.regionCacheRatioOnOldServerMap.get(plan.getRegionName());
+    float ratioThreshold =
+      this.configuration.getFloat(CACHE_RATIO_THRESHOLD, CACHE_RATIO_THRESHOLD_DEFAULT);
```

Comment:
This could be a pretty hot path on a bigger, imbalanced cluster. So I wonder if we should move this conf lookup to somewhere less hot?

Reply:
Moved this and the sleepTime to the loadConf() method